modules/jenkins-webhook-relay/logicapp.tf

```diff
@@ -0,0 +1,29 @@
+resource "azurerm_logic_app_workflow" "logic_app_workflow" {
```

Comment:
could we not duplicate this module and put it in a separate repo?

Reply:
i was originally working on repo https://github.com/hmcts/github-jenkins-integration.  i will create a new repo by name terrafrom-module-jenkins-webhook-relay